### DISCUSSION TITLE
web hook for posting in telegrams

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#4087] [Q&A]
Debian 10, Rolling Release # ccdee79
I ask for help, I set up a web hook for posting in telegrams but nothing is sent, I inserted the token, I registered it at the shit bot, the answer is 200
*******
Debug token changed:
***********
`Custom API base url gave me @Manybot main bot https://core.telegram.org/bots/api
HTTP client POST call to https://core.telegram.org/bots/api/bot123456789:AAELtqcuFwmEfZN7MGsONINdcbiubibywcbklU/sendMessage produced response 200 `
*****************
Debug token changed:Webhook telegram returned code 200
****************
`request_url
"https:\/\/core.telegram.org\/bots\/api\/bot123456789:AAELtqcuFwmEvsdvvwJNJGHSePu6lwY0dR1XklU\/sendMessage"
request_params
{
    "chat_id": "@mybot",
    "text": "\u0421\u0435\u0439\u0447\u0430\u0441 \u0438\u0433\u0440\u0430\u0435\u0442 \u043d\u0430 radio.com: Mulholland Bound - Dam Nation! \u041f\u043e\u0434\u043a\u043b\u044e\u0447\u0430\u0439\u0442\u0435\u0441\u044c \u0441\u0435\u0439\u0447\u0430\u0441.",
    "parse_mode": "Markdown"
}`
**********
`response_body
"<!DOCTYPE html>\n<html class=\"\">\n  <head>\n    <meta charset=\"utf-8\">\n    <title>api\/bot123456789:AAELtqcuFwSDNISBIBBbbbbkl;swY0dRf1XklU\/sendMessage<\/title>\n    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\n    <meta property=\"description\" content=\"\">\n    <meta property=\"og:title\" content=\"api\/bot123456789:AAELtqcuFwsdcinoNNUUOP[PSD0dR1XklU\/sendMessage\">\n    <meta property=\"og:image\" content=\"\">\n    <meta property=\"og:description\" content=\"\">\n    <link rel=\"shortcut icon\" href=\"\/favicon.ico?4\" type=\"image\/x-icon\" \/>\n\n    <link href=\"\/css\/bootstrap.min.css?3\" rel=\"stylesheet\">\n    \n    <link href=\"\/css\/telegram.css?212\" rel=\"stylesheet\" media=\"screen\">\n    <style>\n    <\/style>\n  <\/head>\n  <body class=\"preload\">\n    <div class=\"dev_page_wrap\">\n      <div class=\"dev_page_head navbar navbar-static-top navbar-tg\">\n        <div class=\"navbar-inner\">\n          <div class=\"container clearfix\">\n            <ul class=\"nav navbar-nav navbar-right hidden-xs\"><li class=\"navbar-twitter\"><a href=\"https:\/\/twitter.com\/telegram\" target=\"_blank\" data-track=\"Follow\/Twitter\" onclick=\"trackDlClick(this, event)\"><i class=\"icon icon-twitter\"><\/i><span> Twitter<\/span><\/a><\/li><\/ul>\n            <ul class=\"nav navbar-nav\">\n              <li><a href=\"\/\/telegram.org\/\">Home<\/a><\/li>\n<li class=\"hidden-xs\"><a href=\"\/\/telegram.org\/faq\">FAQ<\/a><\/li>\n<li class=\"hidden-xs\"><a href=\"\/\/telegram.org\/apps\">Apps<\/a><\/li>\n<li class=\"\"><a href=\"\/api\">API<\/a><\/li>\n<li class=\"\"><a href=\"\/mtproto\">Protocol<\/a><\/li>\n<li class=\"\"><a href=\"\/schema\">Schema<\/a><\/li>\n            <\/ul>\n          <\/div>\n        <\/div>\n      <\/div>\n      <div class=\"container clearfix\">\n        <div class=\"dev_page\">\n          <div id=\"dev_page_content_wrap\" class=\" \">\n  <div class=\"dev_page_bread_crumbs\"><\/div>\n  <h1 id=\"dev_page_title\">api\/bot123456789:AAELtqcuFwmJHYYBIOPMKMKMMACePu6lwY0dR1XklU\/sendMessage<\/h1>\n  \n  <div id=\"dev_page_content\">The page has not been saved<\/div>\n  \n<\/div>\n          \n        <\/div>\n      <\/div>\n      <div class=\"footer_wrap\">\n  <div class=\"footer_columns_wrap footer_desktop\">\n    <div class=\"footer_column footer_column_telegram\">\n      <h5>Telegram<\/h5>\n      <div class=\"footer_telegram_description\"><\/div>\n      Telegram is a cloud-based mobile and desktop messaging app with a focus on security and speed.\n    <\/div>\n\n    <div class=\"footer_column\">\n      <h5><a href=\"\/\/telegram.org\/faq\">About<\/a><\/h5>\n      <ul>\n        <li><a href=\"\/\/telegram.org\/faq\">FAQ<\/a><\/li>\n        <li><a href=\"\/\/telegram.org\/blog\">Blog<\/a><\/li>\n        <li><a href=\"\/\/telegram.org\/jobs\">Jobs<\/a><\/li>\n      <\/ul>\n    <\/div>\n    <div class=\"footer_column\">\n      <h5><a href=\"\/\/telegram.org\/apps#mobile-apps\">Mobile Apps<\/a><\/h5>\n      <ul>\n        <li><a href=\"\/\/telegram.org\/dl\/ios\">iPhone\/iPad<\/a><\/li>\n        <li><a href=\"\/\/telegram.org\/dl\/android\">Android<\/a><\/li>\n        <li><a href=\"\/\/telegram.org\/dl\/wp\">Windows Phone<\/a><\/li>\n      <\/ul>\n    <\/div>\n    <div class=\"footer_column\">\n      <h5><a href=\"\/\/telegram.org\/apps#desktop-apps\">Desktop Apps<\/a><\/h5>\n      <ul>\n        <li><a href=\"\/\/desktop.telegram.org\/\">PC\/Mac\/Linux<\/a><\/li>\n        <li><a href=\"\/\/macos.telegram.org\/\">macOS<\/a><\/li>\n        <li><a href=\"\/\/telegram.org\/dl\/web\">Web-browser<\/a><\/li>\n      <\/ul>\n    <\/div>\n    <div class=\"footer_column footer_column_platform\">\n      <h5><a href=\"\/\">Platform<\/a><\/h5>\n      <ul>\n        <li><a href=\"\/api\">API<\/a><\/li>\n        <li><a href=\"\/\/translations.telegram.org\/\">Translations<\/a><\/li>\n        <li><a href=\"\/\/instantview.telegram.org\/\">Instant View<\/a><\/li>\n      <\/ul>\n    <\/div>\n  <\/div>\n  <div class=\"footer_columns_wrap footer_mobile\">\n    <div class=\"footer_column\">\n      <h5><a href=\"\/\/telegram.org\/faq\">About<\/a><\/h5>\n    <\/div>\n    <div class=\"footer_column\">\n      <h5><a href=\"\/\/telegram.org\/blog\">Blog<\/a><\/h5>\n    <\/div>\n    <div class=\"footer_column\">\n      <h5><a href=\"\/\/telegram.org\/apps\">Apps<\/a><\/h5>\n    <\/div>\n    <div class=\"footer_column\">\n      <h5><a href=\"\/\">Platform<\/a><\/h5>\n    <\/div>\n    <div class=\"footer_column\">\n      <h5><a href=\"https:\/\/twitter.com\/telegram\" target=\"_blank\" data-track=\"Follow\/Twitter\" onclick=\"trackDlClick(this, event)\">Twitter<\/a><\/h5>\n    <\/div>\n  <\/div>\n<\/div>\n    <\/div>\n    <script src=\"\/js\/main.js?42\"><\/script>\n    \n    <script>backToTopInit(\"Go up\");\nremovePreloadInit();\n<\/script>\n  <\/body>\n<\/html>\n\n"`

**Comment by BusterNeece** [comment]
@zloysystem Did you change the API URL for the bot? If you want to use the standard Telegram bots, you should leave that field empty (it defaults to `https://api.telegram.org`).

**Reply by ghost** [reply]
> @zloysystem Did you change the API URL for the bot? If you want to use the standard Telegram bots, you should leave that field empty (it defaults to `https://api.telegram.org`).

bot exists, and the configuration contains @MYBOT
Webhook telegram returned code 400

response_body
"{"ok":false,"error_code":400,"description":"Bad Request: chat not found"}"

**Reply by radiopulsar** [reply]
yes, same with me #4136

**Comment by ghost** [comment]
@SlvrEagle23 please tell me where you can change the value of the number of previously played tracks displayed from 15 to a larger value
